pandas/core/indexing.py

```diff
@@ -1356,21 +1341,8 @@ def _validate_read_indexer(
 
             ax = self.obj._get_axis(axis)
 
-            # We (temporarily) allow for some missing keys with .loc, except in
-            # some cases (e.g. setting) in which "raise_missing" will be False
-            if raise_missing:
-                not_found = list(set(key) - set(ax))
-                raise KeyError(f"{not_found} not in index")
-
-            not_found = key[missing_mask]
-
-            with option_context("display.max_seq_items", 10, "display.width", 80):
-                raise KeyError(
-                    "Passing list-likes to .loc or [] with any missing labels "
-                    "is no longer supported. "
-                    f"The following labels were missing: {not_found}. "
-                    "See https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#deprecate-loc-reindex-listlike"  # noqa:E501
-                )
+            not_found = list(set(key) - set(ax))
```

Comment:
we *may* wan to limit the display here to max_seq_items (e.g. if you have a huge list), but could investigate if this is actually an issue. it *can* be potentially if it takes time to render and for example we are NOT raising this to the user (that's not true in this case, speaking more generally here).

Reply:
Yeah this is a good point. Will try to investigate, if we can do this when we supress the error.